specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2019-11-01-preview/appconfiguration.json

```diff
@@ -939,6 +939,25 @@
         "encryption": {
           "$ref": "#/definitions/EncryptionProperties",
           "description": "The encryption settings of the configuration store."
+        },
+        "privateEndpointConnections": {
+          "description": "The list of private endpoint connections that are set up for this resource.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnectionReference"
+          }
+        },
+        "publicNetworkAccess": {
```

Comment:
Also this one. Adding non-readonly property to response is considered breaking change.

Reply:
@NullMDR Public network access needs to be editable. We haven't made a breaking change before but we thought we would be able to since our API version is preview. Please let me know if that is not the case.

Reply:
@NullMDR This API needs to be editable and we are under the assumption that we can fix this because this is a preview version. Please let me know if this is not the case. Thanks!